macroslib/src/typemap/parse.rs

```diff
@@ -724,7 +724,7 @@ impl VisitMut for FilterSwigAttrs {
             .to_string()
             .starts_with("swig_")
         {
-            *i = syn::parse_quote! { #[doc = ""] };
```

Comment:
This fixes "warning: empty doc comment" caused by clippy::empty_docs
https://rust-lang.github.io/rust-clippy/master/index.html#empty_docs